pkg/e2e/routemonitors/routemonitors.go

```diff
@@ -134,10 +139,11 @@ func Create() (*RouteMonitors, error) {
 	}
 
 	return &RouteMonitors{
-		Monitors:  make(map[string]<-chan *vegeta.Result, 0),
-		Metrics:   make(map[string]*vegeta.Metrics, 0),
-		Plots:     make(map[string]*plot.Plot, 0),
-		targeters: targeters,
+		Monitors:   make(map[string]<-chan *vegeta.Result, 0),
+		Metrics:    make(map[string]*vegeta.Metrics, 0),
+		Plots:      make(map[string]*plot.Plot, 0),
+		targeters:  targeters,
+		ReportData: make(map[string][][]float64, 0),
```

Comment:
```suggestion
		Monitors:   make(map[string]<-chan *vegeta.Result),
		Metrics:    make(map[string]*vegeta.Metrics),
		Plots:      make(map[string]*plot.Plot),
		targeters:  targeters,
		ReportData: make(map[string][][]float64),
```

Maps don't need to be initialized with a size, unlike slices. They're backed by some kind of fancy b-tree rather than an array.

Reply:
Got it. I've removed the size parameters. Thanks!